### ISSUE TITLE
Guifont undefined breaks plugin `option.lua:22`

### ISSUE THREAD

**Issue by paginabianca** [SergioRibera/codeshot.nvim#1]
Awesome plugin :rocket: 

Plugin still breaks at `option.lua:22` when `vim.opt.guifont` is not set.
```
Failed to run `config` for codeshot.nvim
...al/share/nvim/lazy/codeshot.nvim/lua/codeshot/option.lua:22: attempt to call method 'replace' (a nil value)
# stacktrace:
  - /codeshot.nvim/lua/codeshot/option.lua:22
  - /codeshot.nvim/lua/codeshot/init.lua:1
  - lua/plugins/codeshot.lua:2 _in_ **setup**
  - lua/plugins/plugins.lua:199 _in_ **config**
Command `SSFocused` not found after loading `codeshot.nvim`
```

Installed with `lazy.nvim`, commit: #4931944

**Comment by SergioRibera** [comment]
Hello, thank you very much for using my plugin 😃 

I have noticed this problem for some time, but I have not had the time to fix it, would you like to make a contribution? anyway, I will try to fix it this week, have a nice day.

**Comment by SergioRibera** [comment]
@paginabianca Solved on #4
